Spigot-Server-Patches/0543-Always-track-EnderDragon.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
+Date: Fri, 12 Jun 2020 21:18:33 -0400
+Subject: [PATCH] Always track EnderDragon
+
+
+diff --git a/src/main/java/org/spigotmc/TrackingRange.java b/src/main/java/org/spigotmc/TrackingRange.java
+index 43b5ed8e396e5312f7de1f160f596f58baead28a..2ba1c1b9160e8d24cb338fa9b6b844423119dc73 100644
+--- a/src/main/java/org/spigotmc/TrackingRange.java
++++ b/src/main/java/org/spigotmc/TrackingRange.java
+@@ -2,6 +2,7 @@ package org.spigotmc;
+ 
+ import net.minecraft.server.Entity;
+ import net.minecraft.server.EntityEnderDragon; // Paper
++import net.minecraft.server.WorldServer; // Paper
+ import net.minecraft.server.EntityExperienceOrb;
+ import net.minecraft.server.EntityGhast;
+ import net.minecraft.server.EntityItem;
+@@ -45,7 +46,7 @@ public class TrackingRange
+             return config.miscTrackingRange;
+         } else
+         {
+-            if (entity instanceof EntityEnderDragon) return defaultRange; // Paper - enderdragon is exempt
++            if (entity instanceof EntityEnderDragon) return ((WorldServer)(entity.getWorld())).getChunkProvider().playerChunkMap.getLoadViewDistance(); // Paper - enderdragon is exempt
```

Comment:
Isn't this meant to be no-tick-view?
> if the view distance or **no-tick view distance** is set higher than 10, the enderdragon can be invisible even though the chunks it is in are visible.
(emphasis mine)

Reply:
load is no tick + 1